src/button/test/clickEvent.test.ts

```diff
@@ -1,37 +1,43 @@
-import { expect, fixture } from "@open-wc/testing";
+import { elementUpdated, expect, fixture } from "@open-wc/testing";
 import { Button } from "../index";
 
-describe("Function onClick event run successfully", () => {
-  const container = new Button({});
-  container.addEventListener("click", (event: Event) => {
-    container.text = "on click";
-  });
+describe("Button", () => {
+  describe("clickEvent", () => {
+    it("should be triggered click event", async () => {
+      const container = new Button({});
+      container.addEventListener("click", (event: MouseEvent) => {
+        container.text = event.type;
+      });
 
-  it("Function onClick event run successfully", async () => {
-    const el = await fixture(container);
-    const buttonEl = el.querySelector(
-      ".kuc-button__button"
-    ) as HTMLButtonElement;
-    await buttonEl.click();
-    expect(buttonEl.innerText).to.be.equal("on click");
-  });
-});
+      const el = await fixture(container);
+      const buttonEl = el.querySelector(
+        ".kuc-button__button"
+      ) as HTMLButtonElement;
 
-describe("Function onClick event not run successfully", () => {
-  const container = new Button({
-    text: "checkOnClick"
-  });
-  container.disabled = true;
-  container.addEventListener("click", (event: Event) => {
-    container.text = "on click";
-  });
+      buttonEl.click();
+      await elementUpdated(buttonEl);
+
+      expect(buttonEl.innerText).to.be.equal("click");
```

Comment:
I think "be" is unnecessary.

```suggestion
      expect(buttonEl.innerText).to.equal("click");
```